route/table.go

```diff
@@ -337,6 +343,16 @@ func (t Table) Lookup(req *http.Request, trace string, pick picker, match matche
 	return target
 }
 
+func (t Table) LookupHostGlob(host string, pick picker) (target *Target) {
```

Comment:
This isn't really a glob matching function since glob matching is already enabled. This only finds https targets.